types/nightwatch/index.d.ts

```diff
@@ -3937,6 +4799,24 @@ export interface WebDriverProtocolSessions {
             result: NightwatchCallbackResult<Array<'client' | 'driver' | 'browser' | 'server'>>,
         ) => void,
     ): this;
+
+    /**
+     * Command to set Chrome network emulation settings.
+     *
+     * @example
+     *  this.demoTest = function() {
+     *    browser.setNetworkConditions({
+     *      offline: false,
+     *      latency: 50000,
+     *      download_throughput: 450 * 1024,
+     *      upload_throughput: 150 * 1024
+     *    });
+     *  };
+     */
+    setNetworkConditions(spec: {
+        [key: string]: any;
+        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<null>) => void;
```

Comment:
the callback function is **not** part of the object: https://nightwatchjs.org/api/setNetworkConditions.html